script.js

```diff
@@ -1,31 +1,121 @@
 // an array with all of our cart items
+
+
 var cart = [];
+// saveToLocalStorage();
+
 
 var updateCart = function () {
   // TODO: finish
+  $('.cart-list').find('div').remove();
+
+  for (var i = 0; i < cart.length; i++) {
+    var carro = cart[i];
+    console.log(carro);
+  }
+  var source = $('#result-template').html();
+
+  var template = Handlebars.compile(source);
+
+  var newHTML = template({cart});
+
+  $('.cart-list').append(newHTML);
+
+  $(".total").html(total);
+
+
+  total();
+}
+
+// var handlebarIt = function (item) {
+  
+//     var source = $('#result-template').html();
+
+//     var template = Handlebars.compile(source);
+
+//     var newHTML = template(item);
+
+//     $('.cart-list').append(newHTML);
```

Comment:
In general, you don't want to leave code commented out. If you want to save past "versions" you can do branches and commits.